jenkins/ci.suse.de/pipelines/socok8s-osh-from-rpm.Jenkinsfile

```diff
@@ -0,0 +1,184 @@
+                    // zypper --gpg-auto-import-keys refresh
+                    // zypper install -yl podman
+                    // usermod --add-subuids 65536-362144 jenkins
+                    // usermod --add-subgids 65536-362144 jenkins
+
+                    // Check if podman is installed
+                    sh "which podman"
+
+                    // Pull the latest leap-15 container image
+                    sh "podman pull registry.opensuse.org/opensuse/leap/15.0/images/totest/images/opensuse-leap-15.0:current"
+
+                    // Write env information to file
+                    // use lowercase SOCOK8S_ENVNAME. CaaSP Velum doesn't like it otherwise
+                    // NOTE(jhesketh): ENVNAME cannot be longer than 28-chars or else it will be truncated and cause errors (build number included)
+                    sh 'echo SOCOK8S_ENVNAME="cloud-socok8s-rpm-${BUILD_NUMBER}" > jenkins.env'
+                    sh 'echo OS_CLOUD="engcloud-cloud-ci" >> jenkins.env'
+                    sh 'echo KEYNAME="engcloud-cloud-ci" >> jenkins.env'
+                    sh 'echo DELETE_ANYWAY="YES" >> jenkins.env'
+                    sh 'echo SOCOK8S_DEVELOPER_MODE="False" >> jenkins.env'
+                    sh 'echo SOCOK8S_USE_VIRTUALENV="False" >> jenkins.env'
+                    sh 'echo DEPLOYMENT_MECHANISM="openstack" >> jenkins.env'
+                    sh 'echo ANSIBLE_STDOUT_CALLBACK="yaml" >> jenkins.env'
+                    sh 'echo USER="root" >> jenkins.env'
+
+                    // Start container
+                    env.CONTAINER_ID = sh(
+                        returnStdout: true,
+                        script: "podman run --env-file jenkins.env --rm -dt registry.opensuse.org/opensuse/leap/15.0/images/totest/images/opensuse-leap-15.0:current"
+                    ).trim()
+                }
+            }
+        }
+
+        stage('Install socok8s package') {
+            steps {
+                script {
+                    // FIXME(jhesketh): Add in Cloud:Rocky repo which has
+                    //                  up-to-date openstacksdk as a temporary
+                    //                  work-around for
+                    ///                 https://bugzilla.suse.com/show_bug.cgi?id=1137590
+                    //                  (Alternatively could install a newer
+                    //                  openstacksdk from pip or elsewhere).
+                    in_container('zypper addrepo http://download.opensuse.org/repositories/Cloud:/OpenStack:/Rocky/openSUSE_Leap_15.0/ "Cloud OpenStack Rocky"')
+                    // CA is required for accessing engcloud
+                    in_container('zypper addrepo http://download.suse.de/ibs/SUSE:/CA/openSUSE_Leap_15.0/ "SUSE CA"')
+                    // Add socok8s repo
+                    in_container('zypper addrepo https://download.opensuse.org/repositories/Cloud:/socok8s/openSUSE_Leap_15.0/ "SUSE OpenStack Cloud on Kubernetes Preview"')
```

Comment:
Hm, this adds fixed version of a socok8s package that's actually TP, right?

I guess we should add ` Cloud:socok8s:master` repo ... and yet not even this is rebuilt  automatically when new sources appear in git, am I right?